### ISSUE TITLE
HUD and GUI elements rubberband when an in-world tooltip is drawn

### ISSUE THREAD

**Issue by ChloeDawn** [baileyholl/Ars-Nouveau#671]
**Mod version:** [1.18.2-2.7.6](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau/files/3944414)

When looking at a block such as a source jar, the player's HUD slides to the right and back momentarily as the tooltip for its amount is drawn. This can also be seen in the inventory menu if the interaction is timed correctly. I'm unsure what the root cause is but I suspect there is a leaked pose stack somewhere.

**Comment by baileyholl** [comment]
Yeah this was added with the new tooltip system, likely somewhere in there.
